packages/casl-ability/src/utils.js

```diff
@@ -1,3 +1,17 @@
 export function wrapArray(value) {
   return Array.isArray(value) ? value : [value];
 }
+
+export function getSubjectName(subject) {
+  if (!subject || typeof subject === 'string') {
+    return subject;
+  }
+
+  const Type = typeof subject === 'object' ? subject.constructor : subject;
+
+  if (Type === Object) {
```

Comment:
I think this is redundant and needs to be deleted.

**Explanation**:
I have been thinking some time ago about possibility to use CASL for cases when only checks per actions are important. So, in that case somebody can define rules for `Object`

```js
AbilityBuilder.define(can => {
   can('read', Object, { public: true })
   can('update', Object, { isMy: true }) 
})
```
So, then it's possible to do something like this:

```js
ability.can('read', { public: true })
```